.github/workflows/rust.yml

```diff
@@ -29,7 +29,7 @@ env:
 
 jobs:
   linux-build:
-    runs-on: buildjet-8vcpu-ubuntu-2204
+    runs-on: "ubuntu-24.04"
```

Comment:
The free OSS runner is 4X runner. build speed from 19mins on buildjet to 21 mins on 4x runner.